src/test/fuzz/merkle.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2025 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <consensus/merkle.h>
+#include <test/fuzz/fuzz.h>
+#include <test/fuzz/FuzzedDataProvider.h>
+#include <test/fuzz/util.h>
+#include <test/util/str.h>
+#include <util/strencodings.h>
+#include <logging.h>
```

Comment:
Why does it need `logging.h`?

Reply:
thanks! 

removed in [c5245d1](https://github.com/bitcoin/bitcoin/pull/32243/commits/c5245d1a76928c70cc25347bbf790246a98912a9)